hardware/v6/README.md

```
@@ -0,0 +1,70 @@
+
+### microlab
+<table><tr>
+<td valign=top><img src="././microlab.svg" width="200" height="200"></td>
+<td valign=top>Parameters:<br/><ul><li>profile: <ul><li>regular</li><li><b>flat</b></li></ul></li></ul></td>
+</tr></table>
+
+## Parts
+
+### control-box
+<table><tr>
+<td valign=top><img src="././control-box.svg" width="200" height="200"></td>
```

Markdownlint (0.37.0)

Images should have alternate text (alt text)

(MD045, no-alt-text)